template.yaml

```diff
@@ -129,6 +139,44 @@ Resources:
             Path: /
             Method: POST
 
+  DataciteDraftDoiHandlerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: datacite-draft-doi-handler
+      Handler: no.unit.nva.datacite.DraftDoiHandler::handleRequest
+      Runtime: java11
+      MemorySize: 1408
+      Environment:
+        Variables:
+          EVENT_BUS: !Ref EventBusName
+          AWC_ACCOUNT_ID: !Ref AWS::AccountId
+      Events:
+        EventBridgeEvent:
+          Type: EventBridgeRule
+          Properties:
+            EventBusName: !Ref EventBusName
+            Pattern:
+              detail:
+                responsePayload:
+                  type:
+                    - doi.publication
+                  item:
+                    doiRequest:
+                      status:
+                        - APPROVED
+                    doi:
+                      - null
```

Comment:
this should be `[null]`

Reply:
Isn't this already an array with the - in front?